hpsklearn/estimator.py

```diff
@@ -525,6 +527,12 @@ def __init__(self,
         self.fit_increment_dump_filename = fit_increment_dump_filename
         self.use_partial_fit = use_partial_fit
         self.refit = refit
+        self._best_preprocs = ()
+        self._best_ex_preprocs = ()
+        self._best_learner = None
+        self._best_loss = None
+        self._best_iters = None
+	self.n_jobs = n_jobs
```

Comment:
The spacing needs to be fixed here. Also, I don't see n_jobs given as a parameter to this function, possibly something got lost in the rebase?